server/src/main/java/io/crate/execution/ddl/tables/TransportCloseTable.java

```diff
@@ -254,13 +254,32 @@ private static IndexTemplateMetadata closePartitionTemplate(IndexTemplateMetadat
     @Override
     protected ClusterBlockException checkBlock(CloseTableRequest request, ClusterState state) {
         String partition = request.partition();
+        if (partition == null && isEmptyPartitionedTable(request.table(), state)) {
+            return state.blocks().globalBlockedException(ClusterBlockLevel.METADATA_WRITE);
+        }
         String indexName = partition == null ? request.table().indexNameOrAlias() : partition;
+        indexNameExpressionResolver.concreteIndexNames(state, STRICT_INDICES_OPTIONS, indexName);
```

Comment:
This method returns `String[] `but return value is not used - is it by design?

Reply:
ah no, this line should be removed, thx!